app/components/helpers.js

```diff
@@ -0,0 +1,33 @@
+import SCForm from './SCForm';
+import { _ } from 'lodash';
+let err_arr = [];
+
+export function nanErr(fieldValue) {
+  _.isNaN(fieldValue) ? this.state.notANumErr : null;
```

Comment:
This function never returns though. And using `this` inside a function is tricky. I don't think this function has any effects when called.